packages/clay-management-toolbar/src/ClayManagementToolbar.soy

```diff
@@ -416,17 +423,20 @@
 		maxSecondaryItems: int,
 		maxTotalItems: int,
 		primaryItems: list<[
+			data: any,
```

Comment:
hey @jbalsas, is giving error when compiling, can you add this parameter in the other templates above that also declare them?